README.md

```diff
@@ -25,7 +25,8 @@ See [roadmap](https://github.com/orgs/openebs/projects/30).
 Before installing LVM driver please make sure your Kubernetes Cluster
 must meet the following prerequisites:
 
-1. all the nodes must have lvm2 utils installed
+1. all the nodes must have lvm2 utils installed and the dm-snapshot kernel
+   module loaded
```

Comment:
please remove this new line here.

Reply:
I flowed it to the same width as the third paragraph below.

Anyways, moved it to the same line.